openapi/p4pa-process-executions.openapi.yaml

```
@@ -135,6 +135,22 @@ paths:
           required: true
           schema:
             type: "ExportFileStatusEnum"
+        - name: filePath
```

let's align to field name
```suggestion
        - name: filePathName
```